src/FirstSteps/RPG/Adventures/Village.cs

```diff
@@ -0,0 +1,28 @@
+﻿using FirstSteps.RPG.Heroes;
+using System;
+using FirstSteps.RPG.Items;
+
+namespace FirstSteps.RPG.Adventures
+{
+    class Village 
+    {
+        public void Enter(Hero hero, Item item)
+        {
+            if (hero is Human)
+            {
+                Console.WriteLine("\nYou enter to the Village!");
+            }
+            else
+            {
+                Console.WriteLine("You can't enter to the village! Only human can enter!");
+                return;
+            }
+            Console.WriteLine("\nPress any button to try get an item!");
+            Console.ReadLine();
+            Human human = (Human)hero;
+            human.AddItemToBackpack(item); // I tryed to do this with variable private Item item; and add item to human.AddItemToBackPack(item) - doesn't work 
```

Comment:
Ok, so first take a look at the open method in OldChect.
`public static Item Open()` what this method can return?

Reply:
Look at this simple example of returning some value from the other class. The 'Permissions` class returns the value and the 'Game` class use this value to display text. 
![image](https://user-images.githubusercontent.com/10303508/212544080-467107c7-ccc7-4a81-a9b4-07cf0d1a8579.png)
In your case, you have to do exactly the same thing but you have `OldChest` instead of `Permissions`.

Reply:
Maybe at first try to display the name of the `item` instead of adding it to the backpack.